### PR TITLE
Reordered documentation

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | yes
| Translation   | yes
| Fixed tickets | N/A
| License       | MIT

Our documentation should be cleaned. I removed some pages (and merged them in others).
It's a draft for the :gb: documentation only. If it's OK for you, we can clean the :fr: and :de: next.

You can have a preview here: http://doc.wallabag.org/en/reorder-documentation/ (before it was http://doc.wallabag.org/en/master/)

### CODE REVIEW

**Comment by Strubbl** [comment]
Yes, i would keep the German docs analog to the English ones, so that it is really only a translation of the English one. Otherwise some time in future they will differ too much

**Comment by jlnostr** [comment]
I agree with @Strubbl :)

File: docs/fr/user/android.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+Double authentification (2FA)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Actuellement, l'application Android ne supporte la double authentification. Vous devez la désactiver pour que l'application fonctionne correctement.
+
+Limiter le nombre d'articles avec wallabag v2
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Dans votre instance de wallabag, vous pouvez configurer combien d'articles se trouvent dans les flux RSS. Cette option n'existe pas dans wallabag v1, où tous les articles se retrouvent donc dans le flux RSS. So if you set the amount of articles being displayed greater than the number of items being content of your RSS feed, you will only see the number of items in your RSS feed.
```
Comment (nicosomb) [inline docs/fr/user/android.rst]: @tcitworld @j0k3r need help to translate this part. I didn't understand it.

**Comment by nicosomb** [comment]
@Strubbl @jlnostr You can reorder the german parts.

**Comment by Strubbl** [comment]
@nicosomb is it possible for you to name the reordering steps? otherwise i have no overview to what went to where.

**Comment by nicosomb** [comment]
@Strubbl You can do like I did for :fr: translation, having a look here https://github.com/wallabag/wallabag/pull/2550/files I think it's the better way... Sorry.

**Comment by nicosomb** [comment]
@jlnostr @Strubbl can you help me to finish this PR please? :)

**Comment by jlnostr** [comment]
Maybe I'll find some time tomorrow evening, otherwise I could do it on Thursday :)

**Comment by nicosomb** [comment]
FYI, it's the latest version before releasing 2.1.4 😇

**Comment by j0k3r** [comment]
@jlnostr ho hell, why do we suddenly have so many commits with merged PR from elsewhere 😳 😞

**Comment by nicosomb** [comment]
I think you `git pull` this branch with an other one.

**Comment by jlnostr** [comment]
Well, seems I've fucked up with git. Let me check for a moment.

**Comment by jlnostr** [comment]
Fixed :)

File: docs/de/index.rst [hunk 2]
```diff
@@ -15,7 +15,7 @@ Die Hauptdokumentation für diese Applikation ist in einigen Abschnitten organis
 
 Die Dokumentation ist in anderen Sprachen verfügbar :
 
-* `Documentation in english <http://doc.wallabag.org/en/master/>`_
+* `Documentation in Englisch <http://doc.wallabag.org/en/master/>`_
```
Comment (nicosomb) [inline docs/de/index.rst]: No, `in english`, because this sentence is in english, not in german.

**Pull request merged**